services/web/client/source/class/osparc/dashboard/StudyBrowserButtonLoadMore.js

```diff
@@ -44,10 +44,13 @@ qx.Class.define("osparc.dashboard.StudyBrowserButtonLoadMore", {
           const rect = element.getBoundingClientRect();
           const html = document.documentElement;
           return (
+            rect.width > 0 &&
+            rect.height > 0 &&
             rect.top >= 0 &&
             rect.left >= 0 &&
-            rect.bottom <= (window.innerHeight || html.clientHeight) &&
-            rect.right <= (window.innerWidth || html.clientWidth)
+            // a bit of tolerance to deal with zooming factors
+            rect.bottom*0.95 <= (window.innerHeight || html.clientHeight) &&
+            rect.right*0.95 <= (window.innerWidth || html.clientWidth)
           );
```

Comment:
is that really how that works??? there is no isVisible() on a widget element ? that is sad.
I also sometimes had the issue with the debugger window open. I hope this will help.

Reply:
Nope, afaik there no isVisible() :(